openedx/features/enterprise_support/utils.py

```diff
@@ -401,7 +402,11 @@ def is_enterprise_learner(user):
     Returns:
         (bool): True if given user is an enterprise learner.
     """
-    return EnterpriseCustomerUser.objects.filter(user_id=user.id).exists()
+    is_enterprise = EnterpriseCustomerUser.objects.filter(user_id=user.id).exists()
```

Comment:
would a get call be needed to check cache hit to avoid the model lookup? https://docs.djangoproject.com/en/3.1/topics/cache/#django.core.caches.cache.get

Reply:
yikes, left that part out. Updated 👍